packages/firestore/src/platform/node_lite/text_reader.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export * from '../browser_lite/text_reader';
```

Comment:
Strangely, yes. The text_reader was extracted from `node_lite/serializer.ts` file, which imported * from '../browser_lite/serializer';
https://github.com/firebase/firebase-js-sdk/blob/master/packages/firestore/src/platform/node_lite/serializer.ts

This might be a mistake in the original code, needs more digging. 🤔

Reply:
Ok sounds good! This probably _is_ a copy/paste error, but it coincidentally works fine. Let's not subtly fix that bug here.